Extensions/Today/TodayViewController.swift

```diff
@@ -11,72 +11,60 @@ import XCGLogger
+        label.font = UIFont.preferredFont(forTextStyle: .body)
+        label.adjustsFontForContentSizeCategory = true
         imageButton.sizeToFit()
         return imageButton
     }()
 
-    fileprivate lazy var openCopiedLinkButton: ButtonWithSublabel = {
-        let button = ButtonWithSublabel()
-        button.setTitle(String.GoToCopiedLinkLabel, for: .normal)
-        button.addTarget(self, action: #selector(onPressOpenClibpoard), for: .touchUpInside)
-        // We need to set the background image/color for .Normal, so the whole button is tappable.
-        button.setBackgroundColor(UIColor.clear, forState: .normal)
-        button.setBackgroundColor(TodayUX.backgroundHightlightColor, forState: .highlighted)
-        button.setImage(UIImage(named: "copy_link_icon")?.withRenderingMode(.alwaysOriginal), for: .normal)
-        button.label.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: UIFont.preferredFont(forTextStyle: .body).withSize(TodayUX.labelTextSize))
+    fileprivate lazy var openCopiedLinkButton: ImageButtonWithLabel = {
+        let imageButton = ImageButtonWithLabel()
+        imageButton.addTarget(self, action: #selector(onPressOpenCopiedLink), forControlEvents: .touchUpInside)
+        imageButton.label.text = String.GoToCopiedLinkLabel + "\n"
+        let button = imageButton.button
+        button.setImage(UIImage(named: "go-to-copied-link")?.withRenderingMode(.alwaysOriginal), for: .normal)
         button.accessibilityLabel = String.GoToCopiedLinkLabel
         button.accessibilityTraits = .button
-        button.label.textColor = TodayUX.labelColor
-        button.label.tintColor = TodayUX.labelColor
-        button.label.sizeToFit()
-        button.subtitleLabel.textColor = TodayUX.subtitleLabelColor
-        button.subtitleLabel.tintColor = TodayUX.subtitleLabelColor
-        button.subtitleLabel.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: UIFont.preferredFont(forTextStyle: .body).withSize(TodayUX.linkTextSize))
-        button.label.accessibilityLabel = String.CopiedLinkLabelFromPasteBoard
-        button.accessibilityTraits = .none
-        return button
-    }()
-
-    fileprivate lazy var widgetStackView: UIStackView = {
-        let stackView = UIStackView()
-        stackView.axis = .vertical
-        stackView.alignment = .fill
-        stackView.spacing = TodayUX.margin / 2
-        stackView.distribution = UIStackView.Distribution.fillProportionally
-        return stackView
+        let label = imageButton.label
+        label.textColor = TodayUX.labelColor
+        label.tintColor = TodayUX.labelColor
+        label.font = UIFont.preferredFont(forTextStyle: .body)
+        label.adjustsFontForContentSizeCategory = true
+        imageButton.sizeToFit()
+        return imageButton
     }()
 
     fileprivate lazy var buttonStackView: UIStackView = {
```

Comment:
@noorhashem 

Under buttonStackView line 73 change it to the following. 
This is your solution to fixing the bug where left new search button isn't aligned
```
stackView.alignment = .top
```

After adding .top it'll align all images on top and it'll look align

<img width="361" alt="image" src="https://user-images.githubusercontent.com/8919439/88116230-22e96a00-cb86-11ea-9d80-bd3d1b761cc0.png">

Reply:
@noorhashem This way you wouldn't need to add an extra "\n"

Reply:
@nbhasin2 but it looks weird when it's aligned to the top if you increase the text to very large sizes since I fixed the font size and the space keeps on increasing. 😥

This is aligned to top at bigger font sizes :
<img width="301" alt="Screen Shot 2020-07-22 at 1 35 02 AM" src="https://user-images.githubusercontent.com/11700198/88117619-e9335600-cbbb-11ea-8614-22905c046789.png">

This is aligned to center at bigger font sizes: 
<img width="304" alt="Screen Shot 2020-07-22 at 1 36 10 AM" src="https://user-images.githubusercontent.com/11700198/88117650-f6e8db80-cbbb-11ea-8ce7-875f223f736c.png">

Reply:
Fix one thing and another breaks. Will need to compromise on a UX solution